ProfilesMicroService.Application/CQRS/Commands/AddOrUpdatePatientByPatientCommand.cs

```diff
@@ -1,8 +1,11 @@
 ﻿using AutoMapper;
+using MassTransit;
 using MediatR;
+using Microsoft.Extensions.Configuration;
 using ProfilesMicroService.Application.Dto.Patient;
 using ProfilesMicroService.Domain.Entities.Models;
 using ProfilesMicroService.Infrastructure.Repository.Abstractions;
+using SharedModel;
```

Comment:
It makes sense to commit this llibrary also as it's not possible to run the project now .. or publish it to the nuget package in the future